libs/SmartStore/SmartStore/Classes/SFSmartStoreInspectorViewController.m

```diff
@@ -28,7 +28,6 @@
 
 #import "SFSmartStoreInspectorViewController.h"
```

Comment:
I wonder if SmartStore inspector should go to its own window also.
(Right now it's not available in React Native apps because I ran into issues when simply pushing the VC on top of RN's VC).